src/wp-admin/includes/theme.php

```diff
@@ -894,78 +894,89 @@ function customize_themes_print_templates() {
-											/* translators: %s: Theme name. */
-											__( 'There is a new version of %s available, but it does not work with your version of WordPress.' ),
-											'{{{ data.name }}}'
-										);
-										if ( current_user_can( 'update_core' ) ) {
-											printf(
-												/* translators: %s: URL to WordPress Updates screen. */
-												' ' . __( '<a href="%s">Please update WordPress</a>.' ),
-												self_admin_url( 'update-core.php' )
-											);
-										}
-										?>
-									<# } else if ( ! data.updateResponse.compatiblePHP ) { #>
-										<?php
-										printf(
-											/* translators: %s: Theme name. */
-											__( 'There is a new version of %s available, but it does not work with your version of PHP.' ),
-											'{{{ data.name }}}'
-										);
-										if ( current_user_can( 'update_php' ) ) {
-											printf(
-												/* translators: %s: URL to Update PHP page. */
-												' ' . __( '<a href="%s">Learn more about updating PHP</a>.' ),
-												esc_url( wp_get_update_php_url() )
-											);
-											wp_update_php_annotation( '</p><p><em>', '</em>' );
-										}
-										?>
-									<# } #>
-								</p>
-							</div>
+							<?php
+							$update_incompatible = '<h3 class="notice-title">' . __( 'Update Incompatible' ) . '</h3><p><# if ( ! data.updateResponse.compatibleWP && ! data.updateResponse.compatiblePHP ) { #>';
+							$update_incompatible .= sprintf(
+								/* translators: %s: Theme name. */
+								__( 'There is a new version of %s available, but it does not work with your versions of WordPress and PHP.' ),
+								'{{{ data.name }}}'
+							);
+							if ( current_user_can( 'update_core' ) && current_user_can( 'update_php' ) ) {
+								$update_incompatible .= sprintf(
+									/* translators: 1: URL to WordPress Updates screen, 2: URL to Update PHP page. */
+									' ' . __( '<a href="%1$s">Please update WordPress</a>, and then <a href="%2$s">learn more about updating PHP</a>.' ),
+									self_admin_url( 'update-core.php' ),
+									esc_url( wp_get_update_php_url() )
+								);
+								$update_incompatible .= wp_update_php_annotation( '</p><p><em>', '</em>', false );
+							} elseif ( current_user_can( 'update_core' ) ) {
+								$update_incompatible .= sprintf(
+									/* translators: %s: URL to WordPress Updates screen. */
+									' ' . __( '<a href="%s">Please update WordPress</a>.' ),
+									self_admin_url( 'update-core.php' )
+								);
+							} elseif ( current_user_can( 'update_php' ) ) {
+								$update_incompatible .= sprintf(
+									/* translators: %s: URL to Update PHP page. */
+									' ' . __( '<a href="%s">Learn more about updating PHP</a>.' ),
+									esc_url( wp_get_update_php_url() )
+								);
+								$update_incompatible .= wp_update_php_annotation( '</p><p><em>', '</em>', false );
+							}
+							$update_incompatible .= '<# } else if ( ! data.updateResponse.compatibleWP ) { #>'
```

Comment:
```suggestion
							$update_incompatible .= '<# } else if ( ! data.updateResponse.compatibleWP ) { #>';
```
And you would've gotten away with it too, if it hadn't been for that pesky semi-colon!